devchat/config.py

```diff
@@ -141,35 +155,39 @@ def _create_sample_file(self):
                     client=Client.OPENAI,
                     api_key=""
                 ),
-                "openai.com": OpenAIProviderConfig(
+                "openai": OpenAIProviderConfig(
                     client=Client.OPENAI,
                     api_key=""
                 ),
+                "anthropic": AnthropicProviderConfig(
+                    client=Client.ANTHROPIC,
+                    api_key=""
+				),
                 "general": ProviderConfig(
                     client=Client.LITELLM
                 )
             },
             models={
                 "gpt-4": OpenAIModelConfig(
                     max_input_tokens=6000,
-                    provider='devchat.ai',
+                    provider='openai',
                     temperature=0,
```

Comment:
Providers are endpoints, not client libraries. If GPT-4 goes to devchat.ai, the provider is not openai.com. Providers and clients are two layers. That's the correct abstraction. For example, devchat.ai is using openai client.